desktop/core/src/desktop/js/parse/jison/sql/phoenix/structure.json

```diff
@@ -52,7 +52,7 @@
     "../generic/update/update_table.jison",
     "../generic/use/use.jison",
     "../generic/sql_error.jison",
-    "../generic/sql_main.jison",
+    "sql_main.jison",
```

Comment:
What was changed in sql_main.jison? If just the quoting we should probably find a way to inherit and override it instead of a full copy paste

Doesn't the new sql_main contain a tun of invalid instructions now?
https://phoenix.apache.org/language/index.html#select

Reply:
`SchemaQualifiedTableIdentifier
 : RegularOrBacktickedIdentifier
   {
     parser.addTableLocation(@1, [ { name: $1 } ]);
     $$ = { identifierChain: [ { name: $1 } ] };
   }
 | RegularOrBacktickedIdentifier '.' RegularOrBacktickedIdentifier
   {
     parser.addDatabaseLocation(@1, [ { name: $1 } ]);
     parser.addTableLocation(@3, [ { name: $1 }, { name: $3 } ]);
     $$ = { identifierChain: [ { name: $1 }, { name: $3 } ] };
   }
 **| DoubleQuotedValue
   {
     parser.addTableLocation(@1, [ { name: $1 } ]);
     $$ = { identifierChain: [ { name: $1 } ] };
   }
 | DoubleQuotedValue '.' DoubleQuotedValue
   {
     parser.addDatabaseLocation(@1, [ { name: $1 } ]);
     parser.addTableLocation(@3, [ { name: $1 }, { name: $3 } ]);
     $$ = { identifierChain: [ { name: $1 }, { name: $3 } ] };
   }**
 ;`
 The bold one is added.

Reply:
For more info and better understanding, @JohanAhlen can help us.

Reply:
Indeed better to split, what you can do is create a new file, say "quoted_table_identifier.jison", only containing:

```
SchemaQualifiedTableIdentifier
 : DoubleQuotedValue
   {
     parser.addTableLocation(@1, [ { name: $1 } ]);
     $$ = { identifierChain: [ { name: $1 } ] };
   }
 | DoubleQuotedValue '.' DoubleQuotedValue
   {
     parser.addDatabaseLocation(@1, [ { name: $1 } ]);
     parser.addTableLocation(@3, [ { name: $1 }, { name: $3 } ]);
     $$ = { identifierChain: [ { name: $1 }, { name: $3 } ] };
   }
 ;
```

Revert to the generic/sql_main.jison import and add quoted_table_identifier.jison.

Rules that are defined multiple times gets merged by jison.

Reply:
Done! please check is it good or not? @JohanAhlen